browsers/safari.json

```diff
@@ -221,7 +221,7 @@
           "engine_version": "613.3.9"
         },
         "16": {
-          "release_date": "2022-09-12",
+          "release_date": "2023-09-12",
```

Comment:
Same here, the release of 16 wasn't in 2023 but the release notes say so: https://developer.apple.com/documentation/safari-release-notes/safari-16-release-notes @jdatapple

Reply:
This is just a typo. I'll make sure it gets corrected.